### PR TITLE
[Confluence] on left/right fixup after #8222

### PR DESCRIPTION
**Pull request by stefansaraev**
some id has changed. if eventlog is disabled. onleft/right is a bit meh now ;)

@ksooo 
//cc @ronie

### CODE REVIEW

File: addons/skin.confluence/720p/IncludesHomeMenuItems.xml [hunk 1]
```diff
@@ -222,7 +222,7 @@
 			<onclick>SetFocus(9000)</onclick>
 			<onclick>EjectTray()</onclick>
 		</control>
-		<control type="image" id="90126">
+		<control type="image" id="90127">
```
Comment (ksooo) [inline addons/skin.confluence/720p/IncludesHomeMenuItems.xml]: I'm not a skinning expert, but imo this should read 90133 as 90127 is already used by the "system" section. At least, 90133 works fine for me. Just tried.

Comment (stefansaraev) [inline addons/skin.confluence/720p/IncludesHomeMenuItems.xml]: yea. both were 90126 before. but 90133 works for me. too.

**Comment by ksooo** [comment]
@ronie good to go?

**Comment by ronie** [comment]
very odd.. i can confirm the issue, but i have no idea why it's happening.

but yeah, this PR fixes it, so get it in.

**Comment by ksooo** [comment]
Thx guys

**Comment by phil65** [comment]
@ronie: probably because id 90126 existed twice?

**Comment by ronie** [comment]
it already did before the change from ksoooo.
there's also a few other duplicated id's in that file.

**Comment by phil65** [comment]
ok, then sorry for the noise. was just my first guess.

**Pull request merged**